src/models/conv_nn.py

```diff
@@ -1,19 +1,27 @@
 import torch
 from torch import nn
+import torch.nn.init as init
+import torch.nn.functional as F
 
 class ConvNet(nn.Module):   
     def __init__(self):
         super(ConvNet, self).__init__()
 
-        self.cnn_layers = None
+        self.cnn_layers = nn.Sequential(
+        nn.Conv2d(in_channels=3, out_channels=32, kernel_size=8, stride=4), nn.ReLU(),
+        nn.Conv2d(in_channels=32, out_channels=64, kernel_size=4, stride=2), nn.ReLU(),
+        nn.Conv2d(in_channels=64, out_channels=64, kernel_size=3, stride=1), nn.ReLU())
 
-        self.linear_layers = None
+        self.flatten = nn.Flatten()
+        self.linear_layers = nn.Sequential(nn.Linear(in_features=36864, out_features=1024),nn.ReLU(),
+        nn.Linear(in_features=1024, out_features=512),nn.ReLU(),nn.Linear(in_features=512, out_features=3))
```

Comment:
Personally, I think I would indent lines 11-17 - but if you guys prefer it like this then it's okay too :)

Reply:
fixed with `black` 😄